sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SpanProcessor.java

```diff
@@ -99,4 +101,13 @@ default CompletableResultCode shutdown() {
   default CompletableResultCode forceFlush() {
     return CompletableResultCode.ofSuccess();
   }
+
+  /**
+   * Closes this {@link SpanProcessor} after processing any remaining spans, releasing any
+   * resources.
+   */
+  @Override
+  default void close() {
```

Comment:
Do we need a shutdown and a close?

Reply:
I don't think so if the spec is ok with changing names.

Reply:
What was the outcome of the discussion of having the shutdown method be optional? Were people ok with using a language-appropriate idiom if one was available?

Reply:
I think using the name `close` instead of `shutdown` would still qualify as providing `shutdown`. But that aspect was not discussed as it was concluded that generally a shutdown functionality is *not* optional (see https://github.com/open-telemetry/opentelemetry-specification/issues/1288).

Reply:
hm. Is it worth adding a clarification to the spec, or do we think this is within our control as language experts?

Reply:
I think no one will complain about some SDK method name. But a general clarification in the spec (library-guidelines.md maybe) would be good like

> Method/constant/... names defined in the spec SHOULD be adjusted to language naming conventions (snake_case, PascalCase, ...) but otherwise used as-is. As an exception, if a well-established name/naming pattern exists in a language for a particular functionality, it SHOULD be used instead. In general, the names should be choosen to adhere to the spec as closely as possible without being awkward or unnatural in the target language.